src/components/common/form/InputWithAdornment.js

```diff
@@ -21,6 +21,7 @@ const InputText = ({
   adornmentStyle,
   adornmentColor,
   adornmentDisabled = false,
+  iconAlignment = 'right',
```

Comment:
1, what have you decided about both paste + scan QR options ? 
in this case you need to display 2 icon buttons somehow - need to discuss this

2. you have repeated code. optimise it at leas by using useMemo:

```javascript
const adornmentJsx = useMemo(() => showAdornment ? (
  <TouchableOpacity style={[styles.adornment, adornmentStyle]} disabled={adornmentDisabled} onPress={_onPress}>
   <Icon size={normalize(adornmentSize)} color={adornmentColor || inputColor} name={adornment} />
 </TouchableOpacity>
) : null, [styles, adornmentStyle, adornmentDisabled, _onPress, adornmentSize, adornmentColor, inputColor, adornment])
```